sdcm/cluster_aws.py

```diff
@@ -522,7 +522,7 @@ def check_spot_termination(self):
 
     @property
     def is_data_device_lost_after_reboot(self) -> bool:
-        if self.parent_cluster.is_data_volume_on_ebs():
+        if self.parent_cluster.params.get("data_volume_disk_num") > 0:
             return False
         return any(ss in self._instance.instance_type for ss in ("i2", "i3", ))
```

Comment:
@roydahan seem like we don't support i4 for example :)

Reply:
here's a PR that would add it for all instances that support it:
https://github.com/scylladb/scylla-cluster-tests/pull/4949